src/main/java/fruitshop/strategy/ReturnOperation.java

```diff
@@ -0,0 +1,26 @@
+package fruitshop.strategy;
+
+import fruitshop.model.FruitTransaction;
+import fruitshop.model.Storage;
+
+public class ReturnOperation implements OperationHandler {
+    public static final String READ_FILE_PATH = "src/main/resources/reportToRead.csv";
+    private final Storage storage;
+
+    public ReturnOperation(Storage storage) {
+        this.storage = storage;
+    }
+
+    @Override
+    public void handleTransaction(FruitTransaction fruitTransaction) {
+        int newQuantity = fruitTransaction.getQuantity()
+                + storage.getQuantity(fruitTransaction.getFruit());
+        if (newQuantity >= 0) {
```

Comment:
The check for a non-negative new quantity is good, but since this is a return operation, there should not be a scenario where the new quantity is negative. The check seems unnecessary.